_extensions/odoo/pycompat.py

```diff
@@ -1,13 +1,6 @@
 # -*- coding: utf-8 -*-
 import sys
 
-PY2 = sys.version_info[0] == 2
```

Comment:
As the doc builds with python3 and python2 now obsolete, I prefer to delete these instructions.

Reply:
I'd suggest removing the entire file and fixing up the few callsites then.

Reply:
As the function is still useful to handle None and bytes, I suggest to move it at the end of translater.py where all the callsites are.

Reply:
I doubt bytes are still involved in there (might have been a concern in Python 3 but even that's doubtful), and `None` should probably be handled ad-hoc specifically where it's a concern.

I say strip it out and log what's happening where it's called, if one of the items ends up potentially being None (not sure which one would even be that) then it can be appended with `or ''` to specifically handle that case.

Reply:
After checking, the argument is never strictly equal to None. I modified the code to strip the strings.

Reply:
Nice! Is the str() call even useful?

Reply:
The build passes without it...